src/OneSheet/SheetFile.php

```diff
@@ -21,7 +21,7 @@ class SheetFile
      */
     public function __construct()
     {
-        $this->filePath = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid(null, 1) . '.xml';
+        $this->filePath = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid('', 1) . '.xml';
```

Comment:
```suggestion
        $this->filePath = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid('', true) . '.xml';
```

Let's also fix this while we're at it.